test/conftest.py

```diff
@@ -0,0 +1,64 @@
+from multiprocessing import Process
+from time import sleep
+import pytest
+
+from ocrd import Resolver, Workspace, OcrdMetsServer
+from ocrd_utils import pushd_popd, disableLogging, initLogging, setOverrideLogLevel, config
+
+from .assets import assets
+
+CONFIGS = ['', 'pageparallel', 'metscache', 'pageparallel+metscache']
+
+@pytest.fixture(params=CONFIGS)
```

Comment:
I was a bit confused by those configs being tied into the workspace fixture, but they seem to be connected in the OCR-D API (`mets_server_url` in the `Workspace` class), so this probably ok?

Reply:
Exactly, they are factually tied because the workspace references the METS Server if present.